lib/DOMWorld.js

```diff
@@ -70,6 +70,13 @@ class DOMWorld {
     }
   }
 
+  /**
+   * @return {boolean}
+   */
+  _hasContext() {
```

Comment:
Public?

Reply:
i'd keep it private - no need to be public.

Reply:
But isn’t the class private?

Reply:
But isn’t the class private?

Reply:
There are plans to eventually make it public as part of #2671. For this reason the `_setContext` was made private.